modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/index.d.ts

```diff
@@ -174,13 +139,49 @@ interface ObjectDefinition {
 	titleObjectFieldName: string;
 }
 
+interface SelectOption {
+	label: string;
+	value: string;
+}
+
+type APIApplicationUIData = Pick<
+	APIApplicationItem,
+	'baseURL' | 'description' | 'title'
+>;
+
+type APIEndpointUIData = Pick<
+	APIEndpointItem,
+	'description' | 'path' | 'scope'
+>;
+
+type APISchemaUIData = Pick<
+	APISchemaItem,
+	'description' | 'name' | 'mainObjectDefinitionERC'
```

Comment:
@lhine Please, sort it like `'description' | 'mainObjectDefinitionERC' | 'name'`.